packages/otelbin-validation/test/main.test.ts

```diff
@@ -153,6 +153,7 @@ describe.each(enumerateTestCases())('Validation API', (distributionName, release
         })).resolves.toMatchObject({
           status: 200,
           data: {
+            path: 'service/pipelines/traces',
```

Comment:
**Issue:** This path will cause issues for cases where there are named pipelines `traces/foobar`.

**Suggestion:** Let `path` be a `string[]` to avoid any confusion.

Reply:
I do not believe you are actual Ben posting

Reply:
I asked @roshan-gh in Slack about the format and he said "I am getting strings from avj anyhow". I can go either way.

Reply:
That however means that parsing needs to be context sensitive on the frontend side. Example: `service/pipelines/traces/foobar/exporters/otlp` needs to be treated different than `exporters/otlp`

Reply:
Switched to an array in 1db512a